src/Directory.Build.props

```diff
@@ -11,11 +11,16 @@
   </PropertyGroup>
 
   <!-- Test runner configuration -->
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)Common\Test\App.config" CopyToOutputDirectory="PreserveNewest" Condition="'$(IsUnitTestProject)' == 'true'" />
+  </ItemGroup>
+
+  <!-- Code analyzers -->
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="$(MicrosoftCodeAnalysisFXCopAnalyzersVersion)" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="$(MicrosoftCodeAnalysisAnalyzersVersion)" />    
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="$(MicrosoftCodeAnalysisAnalyzersVersion)" />
+    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="$(MicrosoftCodeAnalysisPublicApiAnalyzersVersion)" />
```

Comment:
Do we even need this entry? We should get them via package reference to ‘Roslyn.Diagnostics.Analyzers’ correct?

Reply:
Yes! I don't know if the behavior is intended or if there is a bug but with only the transitive dependency the diagnostics get raised during command line build but not within VS. Adding the analyzer explicitly allows for the mechanism to work properly in VS.